app/models/katello/pool.rb

```diff
@@ -71,6 +71,19 @@ def upstream?
       subscription.present? && subscription.redhat?
     end
 
+    def import_audit_record(old_host_ids, new_host_ids = subscription_facets.pluck(:host_id))
+      return if old_host_ids.empty? && new_host_ids.empty?
+      pool_id = self.id
+      logger.info "Inserting audit record for pool_id:#{pool_id}"
```

Comment:
This isn't needed, we already log audit creation with https://github.com/theforeman/foreman/blob/develop/app/models/concerns/audit_extensions.rb#L11